### ISSUE TITLE
Disable static themes for Android before FF65

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-frontend#7419]
We should flip the `enableFeatureStaticThemesForAndroid` config flag to disable static themes for Android before the FF65 release, which will be released on January 29, 2019 because of the static theme migration.

This flag should be turned on again after the FF65 release.

**Comment by willdurand** [comment]
I updated the issue because the first version did not really make sense.

**Comment by willdurand** [comment]
Given that the FF65 release is in 12 days, we're wondering if that even makes sense to disable ST now...

**Comment by muffinresearch** [comment]
@willdurand For better Android UX we might want another issue to either:

- Ensure a compatibility message is shown on Android to say "Basic static theme support is only available from FFAndroid 65 onwards" and show that to FF Android < 65.
- Hide themes completely from FF Android < 65

My vote would probably be with the first option. We could even promote the link to get FF Android.

**Comment by AlexandraMoga** [comment]
We do seem to have this compatibility message in stage already:


![image](https://user-images.githubusercontent.com/31961530/51312694-85731e80-1a54-11e9-8ea6-967231a93586.png)


In -dev however, we are showing a platform incompatibility message:


![image](https://user-images.githubusercontent.com/31961530/51312700-8ad06900-1a54-11e9-87e1-5c0ad7415770.png)

**Comment by muffinresearch** [comment]
@AlexandraMoga oh that is perfect 👍 The best issues are the ones that are already fixed.

@willdurand On that basis and in light of https://github.com/mozilla/addons-frontend/issues/7419#issuecomment-455118579 I think we can probably wontfix this.

**Comment by muffinresearch** [comment]
Sorry meant to only comment not close it as well :/

**Comment by muffinresearch** [comment]
Would be good to understand the reason for the difference in -dev vs stage though.

**Comment by willdurand** [comment]
According to those values https://github.com/mozilla/addons-server/blob/master/src/olympia/constants/base.py#L409:L413, I think we'll get the correct error messages most of the time (as the one from stage in the comment above).

**Comment by willdurand** [comment]
> I would think that once the migration is complete and settled we can file an issue to remove this flag.

👍
